pytest_helm_charts/utils.py

```diff
@@ -89,3 +97,45 @@ def inject_extra(
     if extra_spec:
         cr_dict["spec"].update(extra_spec)
     return cr_dict
+
+
+def delete_and_wait_for_objects(
+    kube_client: HTTPClient,
+    obj_type: Type[T],
+    objects_to_del: Iterable[T],
+    timeout_sec: int = DEFAULT_DELETE_TIMEOUT_SEC,
+) -> None:
+    for kube_object in objects_to_del:
+        kube_object.delete()
+
+    any_exists = True
+    times = 0
+    while any_exists:
+        if times >= timeout_sec:
+            raise WaitTimeoutError(f"timeout of {timeout_sec} s crossed while waiting for objects to be deleted")
```

Comment:
```suggestion
            raise WaitTimeoutError(f"timeout of {timeout_sec} s exceeded while waiting for objects to be deleted")
```

Crossed is fine but exceeded would also work for this.